server/lib/auth.js

```diff
@@ -15,7 +15,7 @@ export const TOKEN_EXPIRATION_SESSION = daysToSeconds(90);
 
 /** Generate a JWToken with the received parameters */
 export function createJwt(subject, payload, expiresIn) {
-  return jwt.sign(payload, config.keys.opencollective.jwtSecret, { expiresIn, subject });
+  return jwt.sign(payload, config.keys.opencollective.jwtSecret, { expiresIn, subject: String(subject) });
```

Comment:
What was the value and type of `subject` before the operation?

Reply:
It was an integer.